x-pack/plugins/apm/public/components/app/error_group_details/error_sampler/error_sample_detail.tsx

```diff
@@ -156,6 +156,7 @@ export function ErrorSampleDetails({
       transactionId: '',
       waterfallItemId: '',
       detailTab: TransactionTab.timeline,
+      flyoutDetailTab: '',
```

Comment:
Instead of removing unwanted params we might want to not apply `...query` and instead only pass params explicitly.